GitTfs/Commands/Init.cs

```diff
@@ -102,6 +102,13 @@ private void DoGitInitDb()
             }
             globals.Repository = gitHelper.MakeRepository(globals.GitDir);
 
+            if (string.IsNullOrWhiteSpace(globals.Repository.GetConfig("user.name"))
+                && string.IsNullOrWhiteSpace(globals.Repository.GetConfig("user.email")))
+            {
+                globals.Repository.SetConfig("user.name", "git-tfs");
+                globals.Repository.SetConfig("user.email", "placeholder@git-tfs.com");
+            }
```

Comment:
Not fond of too :-(

But after some different tries, that's the solution I found to make
integration tests pass on TeamCity.

This change is only here to set git user data in the repositories create on
TeamCity because it seems that there is no global git config on agents.